sqlserver/assets/configuration/spec.yaml

```diff
@@ -284,6 +284,12 @@ files:
             type: boolean
             example: True
           hidden: true
+        - name: max_queries
```

Comment:
Quick comment: i recommend making this config hidden. The only time a customer would ever need to set this value is in the case when we modify their limits applied on the backend. In that case we'll be working closely with the customer. Having it displayed in the config for all customers will potentially lead to more confusion. 

I would also modify the description to say something to this effect.

Reply:
got it, applied these changes.